Connections/canvas-context-connections.json

```diff
@@ -62,17 +62,167 @@
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "RelatedURLs",
+      "connectionDescription": "Related url",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "CommunicatedHosts",
+      "connectionDescription": "Communicated with host",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "CommunicatedIPs",
+      "connectionDescription": "Communicated with host",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "CommunicatedURLs",
+      "connectionDescription": "Communicated with host",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "CommunicatedDomains",
+      "connectionDescription": "Communicated with domain",
+      "parentContextKey": "File"
+    },
+
+    {
+      "contextKey1": "Name",
+      "contextKey2": "DownloadedHashes.sha256",
+      "connectionDescription": "Hash downloaded from domain",
+      "parentContextKey": "Domain"
+    },
+    {
+      "contextKey1": "Name",
+      "contextKey2": "Resolutions.ip_address",
+      "connectionDescription": "DNS resolution",
+      "parentContextKey": "Domain"
+    },
+    {
+      "contextKey1": "Name",
+      "contextKey2": "CommunicatingHashes.sha256",
+      "connectionDescription": "Hash communicated with domain",
+      "parentContextKey": "Domain"
+    },
+    {
+      "contextKey1": "Name",
+      "contextKey2": "Subdomains",
+      "connectionDescription": "Sub domain",
+      "parentContextKey": "Domain"
+    },
+
+    {
+      "contextKey1": "Address",
+      "contextKey2": "Resolutions.hostname",
+      "connectionDescription": "DNS resolution",
+      "parentContextKey": "IP"
+    },
+    {
+      "contextKey1": "Address",
+      "contextKey2": "DownloadedHashes.sha256",
+      "connectionDescription": "Hash downloaded from IP",
+      "parentContextKey": "IP"
+    },
+    {
+      "contextKey1": "Address",
+      "contextKey2": "CommunicatingHashes.sha256",
+      "connectionDescription": "Hash communicated with IP",
+      "parentContextKey": "IP"
+    },
+    {
+      "contextKey1": "Address",
+      "contextKey2": "DetectedURLs.url",
+      "connectionDescription": "URLs hosted at this IP address that have url scanner positive detections",
+      "parentContextKey": "IP"
+    },
+    {
+      "contextKey1": "Address",
+      "contextKey2": "UnAVDetectedReferrerHashes.sha256",
+      "connectionDescription": "UnAV Detected Referrer Hashes",
+      "parentContextKey": "IP"
+    },
+    {
+      "contextKey1": "Address",
+      "contextKey2": "UnAVDetectedDownloadedHashes.sha256",
+      "connectionDescription": "Downloaded from the IP without AV detection",
+      "parentContextKey": "IP"
+    },
+    {
+      "contextKey1": "Address",
+      "contextKey2": "ReferrerHashes.sha256",
+      "connectionDescription": "Detected by AV",
```

Comment:
Should be "AV Detected Referrer Hashes" to match the information from above in "UnAVDetectedReferrerHashes.sha256"